src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1541,6 +1542,15 @@ std::set< replica_id_t > RaftReplDev::get_active_peers() const {
     return res;
 }
 
+uint32_t RaftReplDev::get_quorum_for_commit() const {
```

Comment:
can I take this as a short cut instead of exposing through nuraft_mesg `int32 raft_server::get_quorum_for_commit()` ?

:)